### PR TITLE
Update to elasticsearch version 7.16.3

### PR DESCRIPTION
**Pull request by TheRealDwright**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 group = org.elasticsearch.plugin.prometheus
 
-version = 7.16.2.1-SNAPSHOT
+version = 7.16.3.0-SNAPSHOT
```
Comment (mindw) [inline gradle.properties:3]:
```suggestion
version = 7.16.3.0
```

Comment (TheRealDwright) [reply gradle.properties]: Previously the repo had been building a snapshot on a PR but this appears to have changed on the last couple of updates.

**Comment by ILunie** [comment]
🙏 🙏 
```java
Exception in thread "main" java.lang.IllegalArgumentException: Plugin [prometheus-exporter] was built for Elasticsearch version 7.13.4 but version 7.16.3 is running
        at org.elasticsearch.plugins.PluginsService.verifyCompatibility(PluginsService.java:391)
        at org.elasticsearch.plugins.cli.InstallPluginAction.loadPluginInfo(InstallPluginAction.java:831)
        at org.elasticsearch.plugins.cli.InstallPluginAction.installPlugin(InstallPluginAction.java:887)
        at org.elasticsearch.plugins.cli.InstallPluginAction.execute(InstallPluginAction.java:245)
        at org.elasticsearch.plugins.cli.InstallPluginCommand.execute(InstallPluginCommand.java:88)
        at org.elasticsearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:77)
        at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:112)
        at org.elasticsearch.cli.MultiCommand.execute(MultiCommand.java:95)
        at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:112)
        at org.elasticsearch.cli.Command.main(Command.java:77)
        at org.elasticsearch.plugins.cli.PluginCli.main(PluginCli.java:36)> (file: /opt/usage/puppet/modules/elasticsearch/manifests/plugin.pp, line: 128)
```

**Review by vvanholl — APPROVED** [review]
LGTM

**Pull request merged**